### ISSUE TITLE
bug in searching for alternatives

### ISSUE THREAD

**Issue by phanirithvij** [lycheeverse/lychee#1501]
This has been running for 10mins
```shell
lychee -E  --cache --scheme http --scheme https --include-verbatim --timeout 5 --suggest
  1659/1659 ━━━━━━━━━━━━━━━━━━━━ Finished extracting links
⠂ 14/15 ━━━━━━━━━━━━━━━━━━ ━ Searching for alternatives   
```
Can the --timeout flag be used for archive checking step as well?

**Comment by mre** [comment]
Great idea. @thomas-zahner fyi

In general, archive.org is super slow quite often at least for me. We could support alternatives, but I'm not aware of any.

**Comment by thomas-zahner** [comment]
Yes, I totally agree that the timeout value should be respected by the archive lookup step.
@mre there aren't many alternatives. archive.today is maybe the only one. There is also the [Time Travel Service](https://timetravel.mementoweb.org/guide/api/) which sounds interesting.

**Comment by mre** [comment]
I'm all for it. Guess we could pass the timeout already. Anyone up for fixing it with a PR?

**Comment by mre** [comment]
Fixed by @thomas-zahner and will be released with the next version 0.6.2.
